flakesmaxobject/flakesmaxobject.vcxproj

```diff
@@ -69,11 +69,11 @@ copy /Y "$(TargetPath)" "$(SolutionDir)..\sandbox\max2017\Current\"
 if not "$(Configuration)" == "Ship" copy /Y "$(TargetDir)\$(TargetName).pdb" "$(SolutionDir)..\sandbox\max2017\$(Configuration)\"
 if not "$(Configuration)" == "Ship" copy /Y "$(TargetDir)\$(TargetName).pdb" "$(SolutionDir)..\sandbox\max2017\Current\"
 
-copy /Y "$(SolutionDir)..\flakesobject\build-$(Configuration)\$(Configuration)\flakesobject.dll" "$(SolutionDir)..\sandbox\max2017\$(Configuration)\"
-copy /Y "$(SolutionDir)..\flakesobject\build-$(Configuration)\$(Configuration)\flakesobject.dll" "$(SolutionDir)..\sandbox\max2017\Current\"
+copy /Y "$(SolutionDir)..\flakesobject\build\$(Configuration)\flakesobject.dll" "$(SolutionDir)..\sandbox\max2017\$(Configuration)\"
```

Comment:
The problem is the following:

Right now, `flakesobject` is built for a particular appleseed DLL (either debug or release), because our `FindAppleseed.cmake` file doesn't yet automatically pick up the right build (debug or release) of the appleseed DLL.